tests/unittests/sources/test_azure.py

```diff
@@ -3759,13 +3770,79 @@ def test_no_pps(self):
         assert len(self.mock_kvp_report_failure_to_host.mock_calls) == 0
         assert len(self.mock_kvp_report_success_to_host.mock_calls) == 1
 
+    @pytest.mark.parametrize("pps_type", ["Savable", "Running"])
```

Comment:
Can we get coverage of pps_type = "None" here too to assert it doesn't generate an error in this case? or is this covered already elsewhere?

Reply:
I think that case is already covered in test_recovery_pps test.